source/_integrations/modbus.markdown

```diff
@@ -1318,6 +1318,13 @@ sensors:
       description: "Suppress values close to zero. If -zero_suppress <= value <= +zero_suppress --> 0. Can be float or integer"
       required: false
       type: float
+    register_size_bytes:
+      description: "Use this field to define a non-standard number of bytes in the request. Only set this value when working when devices that don't fully implement the modbus standard, this values is 2 bytes by the standard.
+      This field is ignored when `data_type: string`, `data_type: int8` or `data_type: uint8`
```

Comment:
This information needs to go in the parameter usage matrix:

https://www.home-assistant.io/integrations/modbus/#parameters-usage-matrix

We do not document the limitations in each parameter (that is simply too much information spread over all the parameters and thus very error prune).